### PR TITLE
[DA-2069] Modify AW2F workflow to write AW2f job run ID

### PR DESCRIPTION
**Pull request by j-kanuch**
## Description of changes/additions
This PR removes the unused `aw2f_file_processed_id` field from the `genomic_set_member` table and PDR schemas. A new field to track the job run was added to the RDR and PDR schemas: `aw2f_job_run_id`. The AW2F generation workflow was updated to record the job run ID.

## Tests
- [x] unit tests

### CODE REVIEW

File: rdr_service/tools/tool_libs/genomic_utils.py [hunk 1]
```diff
@@ -2111,7 +2116,7 @@ def run():
                                            'AW4_WGS_WORKFLOW',
                                            'AW5_ARRAY_MANIFEST',
                                            'AW5_WGS_MANIFEST',
-                                           'AW2F_MANIFEST'
+                                           'AW2F_MANIFEST',
```
Comment (dtharpeuno) [inline rdr_service/tools/tool_libs/genomic_utils.py:2119]: haha, haven't we fixed this?

**Review by dtharpeuno — APPROVED** [review]

File: rdr_service/tools/tool_libs/genomic_utils.py (hunk 1 above)
Comment (j-kanuch) [reply rdr_service/tools/tool_libs/genomic_utils.py:2119]: I keep fixing it locally and then not comitting it.

**Review by pegbertsch — APPROVED** [review]
Approved.   @j-kanuch Can you clarify, is the plan to apply this migration to BigQuery to add the new column before you run a local tool to populate this field?   If so is there a timeframe for when you were going to run the tool?

**Comment by j-kanuch** [comment]
> Approved. @j-kanuch Can you clarify, is the plan to apply this migration to BigQuery to add the new column before you run a local tool to populate this field? If so is there a timeframe for when you were going to run the tool?

@pegbertsch I won't run the tool to populate this field until the migration is complete and this code is on production. The backfill tool will most-likely be developed next sprint.

**Comment by j-kanuch** [comment]
> Approved. @j-kanuch Can you clarify, is the plan to apply this migration to BigQuery to add the new column before you run a local tool to populate this field? If so is there a timeframe for when you were going to run the tool?

@pegbertsch I won't run the tool to populate this field until the migration is complete and this code is on production. The backfill tool will most-likely be developed next sprint.

**Pull request merged**